package.json

```diff
@@ -51,6 +51,7 @@
     "build": "node ./node_modules/webpack/bin/webpack.js",
     "start": "node ./node_modules/webpack-dev-server/bin/webpack-dev-server.js",
     "test": "./node_modules/karma/bin/karma start",
+    "preinstall": "npm install -g typings && typings install",
```

Comment:
i could be wrong, but i _think_ this can be changed to just `"postinstall": typings install`.